### PR TITLE
chore: reduce docker image size

### PR DESCRIPTION
**Pull request by zackpollard**
Reduces machine-learning docker image size from 2.4GB to ~850MB
Reduces server docker image size from 505MB to ~350MB
Reduces web docker image size from 350MB to ~ 150MB

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]
🚀

**Review by jrasm91 — APPROVED** [review]
Nice work 🎉

File: server/package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "cookie-parser": "^1.4.6",
     "diskusage": "^1.1.3",
     "exiftool-vendored": "^19.0.0",
+    "exiftool-vendored.pl": "^12.54.0",
```
Comment (samip5) [inline server/package.json:61]: I hope you are aware that this is not meant to be used directly as a dep?

"You shouldn't include either the exiftool-vendored.exe or exiftool-vendored.pl as direct dependencies to your project." - https://www.npmjs.com/package/exiftool-vendored install section

Comment (jrasm91) [reply server/package.json:61]: Yeah, we looked at this. That works fine, but we are specifically running `npm ci --omit=optional` to decrease the number of unused packages that are installed on the docker image. Unfortunately this is marked as an optional and there really isn't a way around getting it installed without listing it as a dependency or installing all our optional deps.

**Pull request merged**